js-sdk/integrations/react/overview.mdx

```diff
@@ -0,0 +1,12 @@
+---
+id: overiview
+title: Tolgee with React
+sidebar_label: Overview
+description: "Get started with Tolgee's integration for React. Learn how to add localization to your React applications, including support for Next.js and React Native."
+---
+
+[React](https://react.dev/) is a library that lets you build user interfaces for your applications. To provide a localized experience to your users, you can add Tolgee to React.
+
+Tolgee provides an SDK for React, based on the [`@tolgee/web`](../../api/web_package/about) package. Using the SDK [@tolgee/react](https://www.npmjs.com/package/@tolgee/react), you can connect your React application to the Tolgee platform. This SDK enables you to implement [in-context translations](../../in-context), [language detection](../../language#language-detection), interpolation, and other advanced features.
+
+Follow the [installation guide](./installation) to add the Tolgee React SDK to your existing React application. You can also check the [example application](https://github.com/tolgee/react-example) to see how to integrate Tolgee with React. If you are building your application with [Next.js](https://nextjs.org/), follow our [Next.js documentation](./next/introduction) for guidance. For React Native applications, view our [React Native documentation](./react_native) to learn how to use Tolgee.
```

Comment:
The overview is nice, however it is very dense text with not much structure. Could we maybe make it a bit more structure (e.g. using bullet points) so the user can quickly find what he needs?

![Snímek obrazovky 2024-07-08 v 9 31 28](https://github.com/tolgee/documentation/assets/8831144/52e0f3c0-802a-4537-a3d0-c8cbba779bbd)